packages/components/src/DataList/components/DataListLayoutInternal/DataListItems.tsx

```diff
@@ -38,3 +43,36 @@ export function DataListItems<T extends DataListObject>({
     />
   );
 }
+
+function ListItemInternal<T extends DataListObject>({
+  children,
+  item,
+}: {
+  children: JSX.Element;
+  item: T;
+}) {
+  const { selectedItems, actions, onSelectChange } = useDataListContext();
+
+  if (actions) {
+    return (
+      <div
+        className={classNames(styles.selectable, [
+          selectedItems.length > 0 && styles.selected,
+        ])}
```

Comment:
forgot that you could do arrays (or no array) with the `&&` to conditionally add CSS. Majority of the time it is written as 

```ts
classNames(styles.selectable, {
  [styles.selected]: selectedItems.length,
})
```